### ISSUE TITLE
Suggestions for AUR package

### ISSUE THREAD

**Issue by sainnhe** [filiparag/wikiman#4]
I notice that you've removed the dependence for `arch-wiki-docs`. Of course this tool works fine without this package, but it will report a warning:

```
$ wikiman foo
warning: ArchWiki documentation for 'en' does not exist
```

So I think it might be better to add `arch-wiki-docs` to `depends` or `optdepends`.

---

If `~/.config/wikiman/wikiman.conf` does not exist, then this tool will report a warning which is really annoying:

```
warning: configuration file missing, using defaults
```

So maybe you can add a default configuration file to `/etc/wikiman.conf` and add this file to `backup` in PKGBUILD.

**Comment by filiparag** [comment]
Thanks for suggesting these changes. I restored `arch-wiki-docs` dependency and added `/etc/wikiman.conf` default configuration file.
Issue resolved in version [2.5-1](https://github.com/filiparag/wikiman/releases/tag/2.5).

**Comment by Nowaker** [comment]
```
% cat /etc/wikiman.conf
# Sources
sources = man

# Quick search mode (only by title)
quick_search = false

# Raw output (for developers)
raw_output = false

# Manpages language(s)
man_lang = en

# Wiki language(s)
wiki_lang = en

# Show previews in TUI
tui_preview = true

# Viewer for HTML pages
tui_html = w3m
```

Arch docs are no longer listed as a source.
